### ISSUE TITLE
[Bug] [Windows] Full page / layout receiving focus

### ISSUE THREAD

**Issue by rachelkang** [dotnet/maui#1184]
### Description

The full page and/or layout receives screen reader focus. Further investigation is required to determine if this is expected native behavior. If not, we need to fix this so the full page/layout does not receive focus!

### Steps to Reproduce

1. Run MAUI sample on Windows
2. Turn on Narrator
3. Notice the focus goes to the whole pge
4. Navigate to semantics page
5. Notice the focus goes to the StackLayout

### Expected Behavior
Page/layout should not receive screen reader focus by default. OR if this is expected, this expectation should be clarified / made known.

### Actual Behavior
Page/layout receives screen reader focus by default.

### Basic Information

- Version with issue:
- Last known good version:
- IDE:
- Platform Target Frameworks: 
  - UWP:   
- Android Support Library Version: 
- Nuget Packages:
- Affected Devices:

**Comment by rachelkang** [comment]
Closing as I'm no longer able to reproduce this issue.
